lessons/pandoc_markdown/lesson.md

```diff
@@ -0,0 +1,218 @@
+---
+layout: page
+title: Intro to Pandoc and Markdown
+visible: true
+---
+
+<!-- change visible to true if you want it on the site -->
+
+# Intro to Pandoc and Markdown
+
+ - **Authors**: Dano Morrison
+ - **Research field**: Neuroscience
+ - **Lesson topic**: How to use Pandoc and Markdown for writing scientific manuscripts
+ - **Lesson content URL**: <https://github.com/UofTCoders/studyGroup/tree/gh-pages/lessons/pandoc_markdown>
+
+# Intro to Pandoc
+
+Do you enjoy the spartan, academic aesthetic of LaTeX documents but not have the time to learn its unforgiving syntax? Does writing HTML documents by hand make you feel uncool? Do you enjoy automating difficult things? If so, Pandoc is what you're looking for. 
+
+Pandoc is a command line tool that you can use to automatically convert files from markup format to another. With Pandoc, you can write in something easy like Markdown, Microsoft Word, or LibreOffice, and convert it something hard like:
+- HTML
+- Ebook formats
+- LaTeX
+- PDF
+- and many others
+
+# Intro to Markdown
+
+Markdown is the best way to write things for Pandoc, and probably the best way to write things for publishing to the web. If you haven't heard of it, it's a lightweight, shortened version of HTML that uses simplified tags like `* ~ - #` to format documents. It's mostly writing plain text, but with a little practice you can easily implement the most common types of text formatting, like headings, lists, links, images, etc. The veritable bible of Markdown is [Daring Fireball.](https://daringfireball.net/projects/markdown/)
```

Comment:
I'm not sure I'd call Daring Fireball that, since it leaves several small things out or ambiguous, but okay.